api/web3server_marshal.go

```diff
@@ -0,0 +1,97 @@
+package api
+
+import (
+	"encoding/hex"
+	"encoding/json"
+
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-core/action"
+)
+
+type (
+	receiptObjectV2 struct {
+		blockHash       hash.Hash256
+		from            string
+		to              *string
+		contractAddress *string
+		logsBloom       string
+		receipt         *action.Receipt
+	}
+
+	logsObjectV2 struct {
+		blockHash hash.Hash256
+		log       *action.Log
+	}
```

Comment:
why need V2 format? to replace the `logsObject` and `receiptObject` now in web3server.go?